bitcoind-rpc/src/Bitcoin/Core/RPC/Blockchain.hs

```diff
@@ -336,7 +337,11 @@ getBlockFilter :: BlockHash -> BitcoindClient CompactFilter
 
 -- | Returns the header of the block corresponding to the given 'BlockHash'
 getBlockHeader :: BlockHash -> BitcoindClient BlockHeader
-getBlockStats' :: BlockHash -> Maybe [Text] -> BitcoindClient BlockStats
+
+{- | Compute per block statistics for a given window. All amounts are in
+ satoshis.  It won't work for some heights with pruning.
+-}
+getBlockStats :: BlockHash -> Maybe [Text] -> BitcoindClient BlockStats
```

Comment:
Should we have an enum for the field names?

Reply:
That's not a bad idea.  Is it stable across recent versions?

Reply:
I'll check against our supported versions 👍🏼

Reply:
I've made the change and tested against the versions we support, there's something fishy against `bitcoin-core-0.19.1` the tests never complete but I confirmed this is unrelated to these changes as the same happens on `master`, all other versions work just fine, I'll take a deeper look into why that happens.